build-scripts/builder/mac/target/mac

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+arg1=$1
+
+case $arg1 in
+    "setup")
+        echo "setup"
+        curl -s https://artifactory.prd.it.unity3d.com/artifactory/unity-tools-local/utr-standalone/utr --output utr
+        chmod +x ./utr
+        unity-downloader-cli -u 2021.2 -c Editor --fast --wait
```

Comment:
In the yamato job definition you're looping on test_editors but your shell scripts are all hard-coded to 2021.2. This will result in jobs for, e.g. trunk that download and run 2021.2 instead

Reply:
fixed. Now with the job inline, it uses the template variables the same way as everywhere else.